tests/backup/backup_test.go

```diff
@@ -147,10 +147,32 @@ var _ = Describe("{BasicBackupCreateWithRules}", func() {
 				CreateBackupLocation(provider, backup_location_name, BackupLocationUID, CredName, CloudCredUID, bucketName, orgID)
 			}
 		})
+		Step("Creating backup schedule policies", func() {
+			dash.Info("Creating backup schedule policy of type interval")
```

Comment:
A better log state would be:
```
Creating backup interval schedule policy
``` 

What are your thoughts on this?

Reply:
Right
Changed it